infrastructure/ithc.tfvars

```diff
@@ -0,0 +1,3 @@
+capacity = "1"
```

Comment:
Do we need this if we are just using AKS?

Reply:
Probably we do not need, but we have the files for all envs for now. I didn't want to make exception for ithc

Reply:
I confirmed with Praveen, we still need this file because there are references in main.tf for these two variables.
We can't remove them from main.tf for now because these values are used in Demo env (it is still in ASE)

Reply:
👍